ElementaryTasks/Arrays.cs

```diff
@@ -49,5 +49,109 @@ public void SumOfElements()
             }
             Console.WriteLine(sum);
         }
-    }   
+
+        public void MostCommonValue()
+        {
+            int[] numbers = { 11, 10, 12, 13, 14, 13, 16, 13, 15, 4 };
+            int longestOccurance = 0;
+            int mostCommonValue = 0;
+
+            for (int i = 0; i < numbers.Length; i++)
+            {
+                int counter = 0;
+
+                for (int j = 0; j < numbers.Length; j++)
+                {
+                    if (numbers[j] == numbers[i])
+                    {
+                        counter++;
+                    }
+                }
+
+                if (counter > longestOccurance)
+                {
+                    longestOccurance = counter;
+                    mostCommonValue = numbers[i];
+                }
+            }
+            Console.WriteLine(mostCommonValue);
+        }
+
+        public void ArrayReverse()
+        {
+            int[] numbers = { 11, 10, 12, 13, 14, 13, 16, 13, 15, 4 };
+            int[] reverseNumbers = new int[numbers.Length];
+            for (int i = numbers.Length; i > 0; i--)
```

Comment:
you can introduce j varible here like that 
`for (int i = numbers.length, j = 0, ; i > 0; i--, j++)`
and remove reverseIndex variable